app/actions.py

```diff
@@ -1094,11 +1113,10 @@ def scrollToOptimalScrollPosition(self):
         extraRows = maxRows - height
         if extraRows > 0:
             optimalRowRatio = self.program.prefs.editor[u'optimalCursorRow']
-            scrollRow = max(
-                0,
-                min(
-                    len(self.lines) - 1,
-                    top - int(optimalRowRatio * (maxRows - 1))))
+            scrollRow = max(0,
+                            min(
+                                len(self.lines) - 1,
+                                top - int(optimalRowRatio * (maxRows - 1))))
```

Comment:
I'm guessing some of these diffs are from different style settings in yapf. Do these change when using `yapf -i --style google <file>`.
Also, there's a tool at `ci_edit/tools/presub` that should do the yapf on changed files and run pylint that might be helpful.

Reply:
Hmm, the style in this CL is actually nicer than the --style google settings. Maybe we should consider not using --style google.